asteroid/masknn/convolutional.py

```diff
@@ -553,7 +553,8 @@ def _fix_dcu_input_dims(fix_length_mode: Optional[str], x, encoders_stride_produ
             pad_shape = [0, time_prod - time_remainder]
             x = nn.functional.pad(x, pad_shape, mode="constant")
         elif fix_length_mode == "trim":
-            x = x[..., :-time_remainder]
+            pad_shape = [0, -time_remainder]
+            x = nn.functional.pad(x, pad_shape, mode="constant")
```

Comment:
Can you have a look at this? 
Maybe this solves the issue you were mentioning? I didn't find the failure on my end. Checking if it works with torch 1.7

Reply:
The issue was somewhere else, in `trim_x_to_y` if both have the same length, but it's fixed now

Reply:
I finally saw that, thanks.